scripts/cfengine-build-host-setup.cf

```diff
@@ -0,0 +1,205 @@
+body file control
+{
+  inputs => { "$(sys.libdir)/stdlib.cf" };
+}
+# assumes that the operating system has working repo lists and has been updated/upgraded already
+
+bundle agent cfengine_build_host_setup
+{
+  packages:
+    debian_9|debian_10::
+      "python-psycopg2";
+    debian_11|debian_12::
+      "python3-psycopg2";
+    ubuntu_16:: # ubuntu_16 doesn't have systemd-coredump by default?
+      "systemd-coredump";
+    ubuntu_20::
+      "autoconf"; # because on arm ubu20 we need to reconfigure
```

Comment:
```suggestion
      "autoconf" comment => "because on arm ubu20 we need to reconfigure";
```

Reply:
oh there you go! :O I don't generally think of policy in that way. excellent addition.